operators/aqua/0.0.1/manifests/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -4,15 +4,17 @@ metadata:
   name: aqua-operator.v0.0.1
   namespace: placeholder
   annotations:
-    # Setting operatorhub.io/ui-metadata-max-k8s-version annotation automatically
-    # The following is an informative annotation to let its users know that this distribution
-    # uses removed APIs in 1.22 and will not work on K8S clusters 1.22+.
-    # Following the findings for this distribution:
-    # this bundle is using APIs which were deprecated and removed in v1.22. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22. Migrate the API(s) for CRD: (["aquacsps.operator.aquasec.com" "aquadatabases.operator.aquasec.com" "aquaenforcers.operator.aquasec.com" "aquagateways.operator.aquasec.com" "aquascanners.operator.aquasec.com" "aquaservers.operator.aquasec.com"])
-    operatorhub.io/ui-metadata-max-k8s-version: "1.21"
+    # Setting olm.maxOpenShiftVersion automatically
+    # This property was added via an automatic process since it was possible to identify that this distribution uses API(s),
+    # which will be removed in the k8s version 1.22 and OpenShift version OCP 4.9. Then, it will prevent OCP users to
+    # upgrade their cluster to 4.9 before they have installed in their current clusters a version of your operator that
+    # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
+    # distribute solutions which is compatible with Openshift 4.9.
+    # For further information, check the README of this repository.
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'
```

Comment:
@yossig-aquasec, 

Why are you removing `operatorhub.io/ui-metadata-max-k8s-version` and replacing it for ` olm.maxOpenShiftVersion`

Note that:

- `operatorhub.io/ui-metadata-max-k8s-version` is an annotation that is valid only for OperatorHub. It was added automatically because we could check that distribution is using APIs which were removed on Kubernetes 1.22 so that, its users are able to check the max Kubernetes versions where this solution can work. You can define the min Kubernetes versions via the CSV spec minKubeVersion. This info appears in the https://operatorhub.io/operator/aqua/alpha/aqua-operator.v0.0.1:

<img width="384" alt="Screenshot 2022-05-11 at 09 39 39" src="https://user-images.githubusercontent.com/7708031/167807282-d0eda678-a40e-4d3b-b4be-b0e9524e5ca2.png">

- `olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "<OCP value>"}]'` : It is used to block cluster admins upgrade their cluster from OCP the value informed with this version installed to avoid workflow issues. Example: If the Operator versions use CRD v1beta1 it cannot work on OCP 4.9. Then, it avoids workflow issues because a cluster-admin upgrades their OCP cluster from 4.8 to 4.9 with an Operator version that cannot work on 4.9 at all. We should ONLY use `olm.maxOpenShiftVersion` for Operators what will be used on Openshift (it is not useful outside of OCP) and when we know that the version cannot work successfully up the OCP version informed.  ℹ️ For further information see the [documentation](https://docs.openshift.com/container-platform/4.10/operators/operator_sdk/osdk-working-bundle-images.html#osdk-control-compat_osdk-working-bundle-images).

In this way: 
a) Could you please do not have comments saying that something was added automatically when it was not
b) Could you please revert the changes and ensure that the distributions that cannot work on kubernetes cluster > 1.21 will have the `operatorhub.io/ui-metadata-max-k8s-version: "1.21"`
c) If you wish you can use the `olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'` but it means keep both. Also, you need to be aware that what is distributed in this repository does NOT appears in Openshift or OKD. 

If you have been looking to distribute solutions for Openshiftt and OKD then you should be adding your project and bundles into: https://github.com/redhat-openshift-ecosystem/community-operators-prod

For further information about what are the sources (index catalogs) used to build OCP see: https://docs.openshift.com/container-platform/4.6/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs 

/hold

Reply:
@yossig-aquasec could you please fix that? IMO it is a blocker for this PR

c/c @J0zi @mvalarh